packages/gatsby-theme-carbon/src/components/LeftNav/LeftNav.module.scss

```diff
@@ -31,7 +32,31 @@
   .bx--side-nav__items > .bx--side-nav__item > .bx--side-nav__submenu span {
     padding-left: 0 !important;
   }
+
+  //dark
+  .bx--side-nav--website--dark
+    .bx--side-nav__menu[role='menu']
+    a.bx--side-nav__link--current,
+  .bx--side-nav--website--dark
+    .bx--side-nav__menu[role='menu']
+    a.bx--side-nav__link[aria-current='page'],
+  .bx--side-nav--website--dark a.bx--side-nav__link--current {
+    background-color: #282828 !important;
```

Comment:
If we're using important do we need all the extra specificity in the rules? It makes it kinda hard to tell what's being targeted.

Reply:
I just copied the rules directly from what we're using in the shell, but can take a look and see if I can make it simpler.

Reply:
Was able to remove one of the important's, but having issues when removing specificity. Going to leave it for now since its using the same rules are the UI Shell. We will be able to remove this in the future anyway when theming is available.